microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md

```diff
@@ -491,22 +492,17 @@ This object represents the original definition of the widget, found in the PWA m
 
 This object is passed as an argument to listeners of service worker widget events of type `widgetinstall`, `widgetuninstall`, and `widgetresume`.
 
-For the `widgetinstall` and `widgetuninstall` event types, the `widgetEvent` object has the following properties:
+For the `widgetinstall`, `widgetuninstall`, and `widgetresume` event types, the `widgetEvent` object has the following properties:
 
 | Property | Description | Type |
 |:--- |:--- |:--- |
-| `widget` | The widget instance that triggered the event. | [widgetInstance](#widgetinstance-object) |
+| `widget` | The widget instance that triggered the event. | [widget](#widget-object) |
 | `instanceId` | The widget instance ID. | `String` |
-
-For the `widgetresume` event type, the `widgetEvent` object has the following property:
-
-| Property | Description | Type |
-|:--- |:--- |:--- |
 | `hostId` | The widget host ID. | `String` |
 
 #### widgetClickEvent object
 
-This object is passed as an argument to listeners of service worker widget events of type `widgetclick`.
+This object is passed as an argument to listeners of service worker widget events of type `widgetclick`. App windows can be opened in response to the `widgetclick` event using `clients.openWindow()`.
```

Comment:
```suggestion
This object is passed as an argument to listeners of service worker widget events of type `widgetclick`. App windows can be opened in response to the `widgetclick` event, by using `clients.openWindow()`.
```
For easier parsing.
Related: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/using-by-using-with